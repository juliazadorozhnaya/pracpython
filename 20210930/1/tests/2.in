13,18,20
