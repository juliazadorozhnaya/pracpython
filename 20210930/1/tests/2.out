[20, 18, 13]
